packages/store/ts/config/v2/namespaces.ts

```diff
@@ -0,0 +1,9 @@
+export type namespacedTableKeys<store> = "tables" extends keyof store
+  ? keyof {
+      [key in keyof store["tables"] as "namespace" extends keyof store
```

Comment:
```suggestion
      [tableName in keyof store["tables"] as "namespace" extends keyof store
```
I've been trying to use named keys rather than just `key` for clarity